src/component/vedtak-panel/vedtak-panel.tsx

```diff
@@ -15,19 +15,17 @@ interface VedtakPanelProps<T> {
 	children: any;
 }
 
-export function VedtakPanel<T>(props: VedtakPanelProps<T>) {
+export function VedtaklistePanel<T>(props: VedtakPanelProps<T>) {
```

Comment:
Kan vi bytte filnamnet her + for tilsvarande css-fil til `vedtakliste-panel` slik at det samsvarar med komponentnamnet?